cli.js

```diff
@@ -8,6 +8,7 @@ const psList = require('ps-list');
 const numSort = require('num-sort');
 const escExit = require('esc-exit');
 const cliTruncate = require('cli-truncate');
+const portToProc = require('portproc').portToProc;
```

Comment:
Can you extract the CLI in `portproc` into a separate package? I don't want to needlessly have `commander` in the dependency tree. See: https://github.com/sindresorhus/ama/issues/17